config/governments.yml

```diff
@@ -0,0 +1,285 @@
+# This list of governments is generated from Whitehall by running the following
+# code in the Rails console:
+# `puts Government.order(start_date: :desc).map { |g| g.as_json(only: %i[id slug name start_date end_date]) }.to_yaml`
+# This is included until government data is in the Publishing API and will
+# need to be updating each time governments change.
+- id: 56
+  slug: 2015-conservative-government
+  name: 2015 Conservative government
+  start_date: 2015-05-08
+  end_date:
+- id: 1
```

Comment:
These IDs don't correspond to the government dates - something looks a bit off here?

Reply:
I assume what happened is that the created every government from newest to oldest sequentially between 2010 and 2015. Then when the 2015 government was created it was given the newest id.